assigment1/task1.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: UTF-8 -*-
+
+from collections import defaultdict
+
+
+def check_words(first_word, second_word):
+    if len(first_word) != len(second_word):
+        return False
+
+    first_word = first_word.decode('utf-8').lower()
+    second_word = second_word.decode('utf-8').lower()
+
+    diff_coutner = defaultdict(int)
```

Comment:
Try to avoid typos in the variable names. This one is very easy to understand, but typos can create misunderstandings.